.identity/02_application_master_auth.tf

```diff
@@ -0,0 +1,17 @@
+resource "azurerm_role_assignment" "master_terraform_subscription" {
+  scope                = data.azurerm_subscription.current.id
+  role_definition_name = "Reader"
+  principal_id         = azuread_service_principal.master.object_id
+}
+
+resource "azurerm_role_assignment" "master_terraform_storage_account_tfstate_app" {
+  scope                = data.azurerm_storage_account.tfstate_app.id
+  role_definition_name = "Contributor"
```

Comment:
is this the same terraform state shared with the main resources? (in other words: does github actions have access to some sensitive data / secrets contained into this storage?)

Reply:
this storage storage account `tfapp<subscription.name>` will be dedicated to save terraform states that will no need to save secrets or sensitive data.

The terraform state for infrastructure with possible secrets and sensitive data will be `tfinf<subscription.name>`